### PR TITLE
Verbose docker build

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
Normally, when docker build runs it minimizes the output.  This is normally a nice user experience but it is unhelpful when debugging docker build issues, when the full printout would be helpful.

All `snsdemo` and many `nns-dapp` scripts support a `--verbose` option; it would be nice if docker-build did the same.

# Changes
- parse the docker-build command line arguments.
- Support these flags:
  - `--verbose`
  - `--help`
  - `--network`

# Tests
- I have used `--verbose` locally and am happy to report that the output has filled many screens with text.
- I have added tests for `--help` and garbage input to CI.  I have not added a test for `--network` or `--verbose` as it seems that the test run time would outweigh the benefit.
- I have checked that the docker build with no flags, as used for creating mainnet builds, is unaffected by this PR:
Main:
```
ea3ab3f9613c5b4bab4734c52360ed18cbe7f83e9290f1f029a700147c74eee8  assets.tar.xz
355c194be9d43378d8f079165c11ce3b585c1e0d4a5ac7f1517eff0c397fab8f  nns-dapp.wasm
d84a70cfd6ac7b59b0d9e537ca4f5f1f2085635f87a35e76e67603a15dc786d2  sns_aggregator.wasm
FIN
max@sinkpad:~/dfn/nns-dapp/branches/release-tesing (8:09)$ git rev-parse HEAD
ff7d09eb4ef2409e84645855614e40918bac0967
max@sinkpad:~/dfn/nns-dapp/branches/release-tesing (25:38)$ 
```
This PR:
```
ea3ab3f9613c5b4bab4734c52360ed18cbe7f83e9290f1f029a700147c74eee8  assets.tar.xz
355c194be9d43378d8f079165c11ce3b585c1e0d4a5ac7f1517eff0c397fab8f  nns-dapp.wasm
d84a70cfd6ac7b59b0d9e537ca4f5f1f2085635f87a35e76e67603a15dc786d2  sns_aggregator.wasm
FIN
max@sinkpad:~/dfn/nns-dapp/branches/verbose-docker-build (49:39)$ git rev-parse HEAD
358187b3bc877c3019e38ccea87571fbbd6a83a5
max@sinkpad:~/dfn/nns-dapp/branches/verbose-docker-build (51:46)$ 
```

### CODE REVIEW

File: .github/workflows/docker-build.yaml [hunk 1]
```diff
@@ -94,14 +94,23 @@ jobs:
           echo "WASM size:          $wasm_size"
           echo "Max supported size: $max_size"
           (( wasm_size <= max_size )) || { echo "The WASM is too large" ; exit 1 ; }
+  docker-build-cli-flags:
```
Comment (lmuntaner) [inline .github/workflows/docker-build.yaml:97]: What's the benefit of adding a test for `--help` flag?

**Review by lmuntaner — APPROVED** [review]
Just a question, but already approving.

File: .github/workflows/docker-build.yaml (hunk 1 above)
Comment (bitdivine) [reply .github/workflows/docker-build.yaml:97]: It's easy to test and checks that at lest some of the code isn't garbage. :-D

Comment (lmuntaner) [reply .github/workflows/docker-build.yaml:97]: hehe, ok

**Pull request merged**